slack-api-model/src/main/java/com/slack/api/model/event/Event.java

```diff
@@ -5,9 +5,7 @@
 public interface Event extends Serializable {
 
     String getType();
-
     default String getSubtype() {
-        return null;
+        return "";
```

Comment:
Due to this change, the CI builds are failing. As the absence of subtype and having an empty string has a different meaning, this is a breaking change. Please revert this part and modify this pull request to work without the change here.

Reply:
Actually that change was not necessary, it was a leftover from a previous attempt to resolve this. Thanks for reviewing and spotting it.

Reply:
Thanks for your prompt response. As I needed to use my time for other tasks today (+09:00), I will check this PR tomorrow.